docs/cloud-messaging/receive.md

```diff
@@ -409,3 +409,97 @@ experimentalSetDeliveryMetricsExportedToBigQueryEnabled(messaging, true);
 ```
 
 Don't forget to run `yarn build` in order to export the new version of your service worker to the `web` folder.
+
+
+## (Advanced, Optional) Allowing Notification Images on iOS
+
+On Apple devices, in order for incoming FCM Notifications to display images from the FCM payload, you must add an additional notification service extension. This is not a required step.
+
+If you are using FirebaseAuth phone authentication, you must add the Firebase Auth pod to your Podfile.
+
+### Step 1 - Add a notification service extension
+
+- From Xcode top menu go to: File > New > Target...
+- A modal will present a list of possible targets, scroll down or use the filter to select "Notification Service Extension". Press Next.
+- Add a product name (use ImageNotification to follow along), set Language to Objective-C and click Finish.
+- Enable the scheme by clicking Activate.
+
+### Step 2 - Add target to the Podfile
+
+Ensure that your new extension has access to Firebase/Messaging pod by adding it in the Podfile:
+
+- From the Navigator open the Podfile: Pods > Podfile
+- Scroll down to the bottom of the file and add:
+
+```ruby
+target 'ImageNotification' do
+  use_frameworks!
+  pod 'Firebase/Auth' # Add this line if you are using FirebaseAuth phone authentication 
+  pod 'Firebase/Messaging'
+end
+```
+
+- Install or update your pods using pod install from the ios and/or macos directory.
```

Comment:
Is pod install a literal command?  Since ios and macos are paths iiuc, those would also be code font.